src/test/unit/controls/Graph/GraphResize-spec.js

```diff
@@ -380,4 +387,44 @@ describe("Graph - Resize", () => {
             });
         });
     });
+    describe("When removeContainerPadding is set to true", () => {
```

Comment:
Is it possible to have the below scenario?. If yes, Can you include the test for the scenario?

- removeContainerPadding is `true` and `legendPadding` is provided?
- removeContainerPadding is `false` and `legendPadding` is not provided?

Reply:
Yes its possible to have the mentioned scenarios. I have included tests for these scenarios [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/105abac166fb4954484342ba7726ebc4b7ae1e7c).